app/scripts/modules/core/src/presentation/forms/stories/FormikIntro.stories.mdx

```diff
@@ -0,0 +1,117 @@
+import { ArgsTable, Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+
+import { FormikFormField } from '../';
+import {URLComponent} from './SpinFormikStoriesHelper'
+
+<Meta
+  title="Forms/Intro"
+  decorators={[
+    (Story) => (
+      <div style={{ background: '#fff', border: '1px solid #eee', borderRadius: 10, padding: 20, width: 500 }}>
+        <Story />
+      </div>
+    ),
+  ]}
+  parameters={{
+    layout: 'centered',
+  }}
+/>
+
+Spinnaker forms are built upon [Formik](https://formik.org) and provides two key abstractions for easily building forms: `SpinFormik` and `FormikFormField`.
+
+## `SpinFormik`
+
+This component is a thin wrapper around [`Formik`](https://formik.org/docs/api/formik) component. A typical form is created the following way
+
+```jsx
+interface FormValues {
+  field1: string;
+  field2: number;
+}
+
+<SpinFormik<FormValues>
+  initialValues={{ field1: 'foo', field2: 2 }}
+  render={(formikProps) => <YourFormView {...formikProps} />}
+  onSubmit={(values) => {
+    /* handle form submission */
+  }}
+/>
+```
+
+### Props
+
+`SpinFormik` passes all props to `Formik`, so check out the [documentation](https://formik.org/docs/api/formik#reference) for detailed information on props accepted by `Formik`.
```

Comment:
Fine for now, but I'd like us to determine what props are actually being used in Deck and hide or remove the rest from the "supported" api.